CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ Change log
 -----------
 
 * Update the Poky submodule, which reverts previous change [Will]
+* Do not show kernel boot messages on the display for production images [Florin]
```

Comment:
Rebase on top of master so that this merge commit does not show up. I think it's wrong to have it here